docs/sources/k6/next/using-k6/k6-options/reference.md

```diff
@@ -786,6 +786,15 @@ export const options = {
 
 ## No summary
 
+{{< admonition type="caution" >}}
```

Comment:
@heitortsergent I've seen some other deprecation messages with `caution` type, but I'm not sure if it's the right level here. Same for the other one below, under `## Summary mode`.